### PR TITLE
Refactor the `UserInterfacePanel` with MVC in mind

### PR DESCRIPTION
**Pull request by madoar**
This PR refactors the `UserInterfacePanel` class with MVC in mind.
In addition it changes the way changes to the interface are handled (now via bindings) and "fixes" the layout:

grafik

### CODE REVIEW

**Comment by plata** [comment]
Do all 3 bold texts use the same font? It looks like the first one is a bit different. Also, I find it strange that there is a `:` after "Erscheinungsbild" but not after the other ones. I guess it was like this before but as you're on it already, it should be fixed as well.

**Comment by madoar** [comment]
Should all three labels have a `:` afterwards or none?

**Comment by madoar** [comment]
@plata I've now removed the `:` from after `Theme:`

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/setting/control/UserInterfacePanel.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public UserInterfacePanelSkin createSkin() {
+        return new UserInterfacePanelSkin(this);
+    }
+
+    public ObservableList<Theme> getThemes() {
+        return themes;
```
Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/setting/control/UserInterfacePanel.java]: `this` for member variables

Comment (madoar) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/setting/control/UserInterfacePanel.java]: Done

**Pull request merged**